### ISSUE TITLE
Changing the Typ of a Connection changes that for all Connections

### ISSUE THREAD

**Issue by ReallyDarkMatter** [chummer5a/chummer5a#3566]
**Describe the bug**
Changing the Typ of a Connection changes that for all Connections.

**To Reproduce**
1. Go to Connections
2. Change the Typ of a Contact let say to fixxer
3. All other Contacts have their typ also changed to fixxer

Just noticed that after the bug with the missing qualities, same Connection lost their Typ.
Typ was empty. So i started to bring them back to the state they had before

**Expected behavior**

**Screenshots**
![image](https://user-images.githubusercontent.com/12108972/55910252-3ca7ad80-5bde-11e9-85d9-fb7ff3dd7cd2.png)
now i change the first contact:
![image](https://user-images.githubusercontent.com/12108972/55910309-6cef4c00-5bde-11e9-8e56-d5cee3254fec.png)

**Character File**
Checked with 2 Chars on two PC

**Desktop (please complete the following information):**
 - OS: [Windows 10]
 - Version 5.205.34

**Comment by ArchonMegalon** [comment]
Oooops - I guess I did that. Messed around in the last build with it to make it accessible for plugins. I will look into that.

**Comment by ArchonMegalon** [comment]
Actually I don't know how to fix that. Seems to be some kind of DependencyGraph and Databinding thing. What I did change before was make the lstContactArchetypes public static with #3568
But I think that should NOT cause this behaviour, the property Role is not static and I'm really not that much into databinding. Sorry - @chummer5a can you help out here please?

**Comment by chummer5a** [comment]
tl;dr binding to a list has a single binding context that is shared by all subscribers to that binding context. If you need to bind to a shared list, instantiate the datasource with a new BindingSource. Doesn't crop up often, but it's hit me a couple times before as well. Anyway, closed by 
 https://github.com/chummer5a/chummer5a/commit/93467cdb43e65b3406f935138b587615b321290f
